src/tests/ftest/util/test_utils_pool.py

```diff
@@ -897,6 +901,97 @@ def drain(self, rank, tgt_idx=None):
 
         return status
 
+    @fail_on(CommandFailure)
+    def get_acl(self):
+        """Get ACL from a DAOS pool.
+
+        Returns:
+            str: dmg pool get-acl output.
+
+        Raises:
+            CommandFailure: if the control method is API.
+
+        """
+        if self.pool:
+            if self.dmg and self.control_method.value == self.USE_DMG:
```

Comment:
(pylint-no-else-return) Unnecessary "elif" after "return"

Reply:
There are two other states that should be handled:

- `control_method` is `self.USE_DMG` but `self.dmg` is undefined
- `control_method` is neither `self.USE_DMG` or `self.USE_API`

Reply:
This could possibly be better implemented with a method that raises a `CommandException` when:

- `self.control_method.value` is `self.USE_API`
- `self.control_method.value` is `self.USE_DMG` but `self.dmg` is undefined
- `self.control_method.value` is neither `self.USE_DMG` or `self.USE_API`

Then it could be called just before any `self.dmg` command.